frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -55,6 +55,8 @@ const PodRing: React.FC<PodRingProps> = ({ pods, obj, resourceKind, path, impers
 
   const isKnative = _.get(obj, 'metadata.ownerReferences[0].kind') === 'Revision';
 
+  const resourceObj = rc || obj;
```

Comment:
Isn't `rc` required? Is it expected to be null or undefined here?

Reply:
When a DC is created both current and Previous are undefined until an RC is created. In that case `rc` will be undefined. Also, StatefulSets don't have current/previous RC data and if we start to show SS in SideBar `rc` will prop will not be passed. I will make `rc` optional. Thanks for noticing. :)

Reply:
fixed in https://github.com/openshift/console/pull/2496/commits/8ee400e6ae62c6d9de3fe35a290b834415a1bed8